rpm/entry.sh

```diff
@@ -1,11 +1,33 @@
 #!/bin/sh
 # Entrypoint for the build container to create the rpms and yum repodata:
+# Usage: ./entry.sh GOARCH/RPMARCH,GOARCH/RPMARCH,....
 
 set -e
-# Download sources if not already available
-cd /root/rpmbuild/SPECS  && spectool -gf kubelet.spec
 
-/usr/bin/rpmbuild --define "_sourcedir /root/rpmbuild/SPECS/" -bb /root/rpmbuild/SPECS/kubelet.spec
+declare -a ARCHS
 
-mkdir -p /root/rpmbuild/RPMS/x86_64/
-createrepo -o /root/rpmbuild/RPMS/x86_64/ /root/rpmbuild/RPMS/x86_64
+if [ $# -gt 0 ]; then
+  IFS=','; ARCHS=($1); unset IFS;
+else
+  #GOARCH/RPMARCH
+  ARCHS=(
+    amd64/x86_64
+    arm/armhf
```

Comment:
I didn't check these, just said some possible values.
However, turns out to be: arm/armhfp, ref: https://admin.fedoraproject.org/mirrormanager/